redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/ModelDescriptions.java

```diff
@@ -3,27 +3,26 @@
 public final class ModelDescriptions {
 
     public static final String ID = "ID of the resource";
+    public static final String CRN = "CRN of the resource";
+    public static final String CREATED = "Creation time of the resource in long";
```

Comment:
(My comment might have been lost here.) This one is redundant with CREATION_DATE, so just use CREATION_DATE since it matches the field name.